packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -51,14 +49,14 @@ export const Avatar = (props: AvatarProps) => {
   }
 
   const ImgOrFallback =
-    initials && (!avatarExists || error) ? (
+    !avatarExists || error ? (
       <InitialsAvatarFallback {...props} />
     ) : (
       <Image
         elementDescriptor={imageElementDescriptor || descriptors.avatarImage}
         alt={fullName}
         title={fullName}
-        src={src || GRAVATAR_DEFAULT_AVATAR}
+        src={src || (profileImageUrl as string)}
```

Comment:
❓ Why do we need the explicit `as string` type assertion here? Let's see if there's a way to avoid it by correctly narrowing its type before that point.

Reply:
@nikosdouvlis this acted as a fallback but I guess it should be just `src={src || ''}`